packages/components/psammead-caption/src/index.jsx

```diff
@@ -36,6 +36,9 @@ const Caption = styled.figcaption`
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
```

Comment:
Please update the `package.json` version for `psammead-caption` (should be a patch increase). Likewise the `package-lock.json` and `CHANGELOG.md` files for this caption component.